http_check/datadog_checks/http_check/http_check.py

```diff
@@ -245,10 +246,12 @@ def send_status_down(loginfo, message):
             self.log.debug("Connecting to %s" % addr)
 
             if disable_ssl_validation and parsed_uri.scheme == "https" and not ignore_ssl_warning:
+                # Emit a warning if disable_ssl_validation is unset
                 if 'disable_ssl_validation' in instance:
                     self.warning("Skipping SSL certificate validation for {0} based on configuration".format(addr))
```

Comment:
Can we demote this to `self.debug`? A customer complained this is too spammy, I think we can assume if you explicitly set `disable_ssl_validation=True` in the settings you know what you're doing.